client/app/widgets/Comments/__tests__/Comments.spec.jsx

```diff
@@ -86,7 +86,7 @@ describe('Comments', () => {
       expect(
         screen.getByRole('button', { name: 'Submit' }),
       ).toBeInTheDocument();
-      expect(screen.getByRole('link', { name: 'Report' })).toBeInTheDocument();
+      expect(screen.getByRole('link', { name: `Report ${value}` })).toBeInTheDocument();
```

Comment:
So this test reveals that for the case of the Report action, we want to pass in the user's name instead of the comment.